repo/packages/M/minio/6/config.json

```diff
@@ -0,0 +1,80 @@
+{
+    "$schema": "http://json-schema.org/schema#",
+    "properties": {
+        "service": {
+            "type": "object",
+            "description": "Service properties for Minio on DC/OS",
+            "properties": {
+                "name": {
+                    "description": "Name of this service instance",
```

Comment:
Service name is without defaults to force the user to give a unique name for each instance and avoid having many instances with the default service name.